src/api/almaApi.ts

```diff
@@ -10,19 +10,11 @@ export const getAlmaRegistration = async (
   systemControlNumber: string,
   invertedCreatorName: string,
   cancelToken?: CancelToken
-) => {
+): Promise<ApiResponse<AlmaRegistration>> => {
```

Comment:
skjønner ikke hvorfor man skal få warning her, for den har jo åpenbart type:
![bilde](https://user-images.githubusercontent.com/6313468/97711801-d931e900-1abd-11eb-8423-34d4aab18475.png)

det må vel være bra nok med en implisitt type? den har jo like mye type nå som om vi hadde satt det eksplisitt, bare at vi slipper å importere alle typene overalt

Reply:
Ser for meg at vi vil ha det i mange tilfeller, men sånn som det er her virker det bare undøvendig IMO. mulig vi kan  tilpasse det slik vi vil: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md#options

Reply:
ja, jeg heller også mot å ignorere denne warningen.

Reply:
syns vi burde harr det for å unngå any, men tillate tilfeller som dette hvor det egentlig er type. ser ikke ut som at noen av settingsene tillater det heller :/ så da blir det fort å droppe ja

Reply:
^ tok meg friheten og oppdaterte. mulig vi må tillate `any` som parameter også enkelte plasser (se `context` i `reference-helpers.ts`), ellers tror jeg reglene er veldig behjelpelige